source/containers/Form/hooks/formActions.ts

```diff
@@ -0,0 +1,82 @@
+import { increaseCount, decreaseCount } from '../../../helpers/Counter';
+import { replaceMarkdownTextInSteps } from './textReplacement';
+import { FormReducerState } from './useForm';
+
+/**
+ * Action for replacing title markdown in steps.
+ * @param {FormReducerState} state the current state of the form
+ */
+export function replaceMarkdownText(state: FormReducerState) {
+  const { steps, user } = state;
+  const updatedSteps = replaceMarkdownTextInSteps(steps, user);
+  return {
+    ...state,
+    steps: updatedSteps,
+  };
+}
+
+/**
+ * Action for decreasing the form counter.
+ * @param {FormReducerState} state the current state of the form
+ */
+export function decreaseFormCounter(state: FormReducerState) {
+  const { counter } = state;
+  return {
+    ...state,
+    counter: decreaseCount(counter, 0),
+  };
+}
+
+/**
+ * Action for increasing the form counter.
+ * @param {FormReducerState} state the current state of the form
+ */
+export function increaseFormCounter(state: FormReducerState) {
+  const { steps, counter } = state;
+  return {
+    ...state,
+    counter: increaseCount(counter, steps.length),
+  };
+}
+
+/**
+ * Action to run when starting a form.
+ * @param {object} state the current state of the form
+ */
+export function startForm(state: FormReducerState, payload: { callback: () => void }) {
+  // TODO: Pass user input values.
+  payload.callback({});
```

Comment:
I believe that payload callback expects zero arguments but got 1, should be empty.

Reply:
Fixed